Xamarin.PropertyEditing/ViewModels/PanelViewModel.cs

```diff
@@ -88,6 +90,14 @@ public void SetIsExpanded (string group, bool isExpanded)
 			SetIsExpanded (ArrangeMode, group, isExpanded);
 		}
 
+		public Task<Stream> GetIconAsync ()
+		{
+			if (TargetPlatform.IconProvider == null)
```

Comment:
The `TargetPlatform.IconProvider` is never set, so always null. Are the changes supposed just to be back-end or is the visual part missing?

Reply:
There's no test icon provider in the standalone, this is just the support for it.

Reply:
@ermau will a test provider be added to the Stand Alone in a future PR and will a test provider  be needed for Mac, down the line?

Reply:
Yes